blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -342,12 +438,22 @@ namespace blueprint
                 for (int i = 0; i < nargs; ++i)
                     args.push_back(engine->readVarFromDukStack(engine->dukContext, i));
 
+                juce::var result;
+
                 // Now we can invoke the user method with its arguments
-                auto result = std::invoke(helper->callback, juce::var::NativeFunctionArgs(
-                    juce::var(),
-                    args.data(),
-                    static_cast<int>(args.size())
-                ));
+                try
```

Comment:
If this looks alien, I believe that currently if a persisted native function throws an Error (like `setTimeout` on bad arguments), you'd get the `Error.what()` message in the red component and nothing else.  This here is to also attach a stack trace to help find the bad call or other useful info.

Reply:
:+1: this looks good!